src/rust/inetstack/protocols/tcp/established/background/mod.rs

```diff
@@ -13,32 +13,32 @@ use self::{
 use crate::{
     inetstack::protocols::tcp::established::ctrlblk::SharedControlBlock,
     runtime::QDesc,
+    scheduler::Yielder,
 };
 use ::futures::{
     channel::mpsc,
     FutureExt,
 };
 
-pub async fn background<const N: usize>(
-    cb: SharedControlBlock<N>,
-    fd: QDesc,
-    _dead_socket_tx: mpsc::UnboundedSender<QDesc>,
-) {
-    let acknowledger = acknowledger(cb.clone()).fuse();
+pub async fn background<const N: usize>(cb: SharedControlBlock<N>, _dead_socket_tx: mpsc::UnboundedSender<QDesc>) {
```

Comment:
`_dead_socket_tx` is not used, then why is it passed into this function?

Reply:
I don't know. :) I thought perhaps we would use it someday because it was included in the first place but I'm not actually sure of the purpose.